modules/snowflake-account/variables.tf

```diff
@@ -31,6 +31,12 @@ variable "snowflake_admin_username" {
   description = "Snowflake admin username created with the initial account subscription."
 }
 
+variable "snowflake_role_name" {
+  type        = string
+  default     = "ACCOUNTADMIN"
+  description = "The name of the role being granted"
+}
+
```

Comment:
ACCOUNTADMIN is a hardcoded role in Snowflake. It is not configurable and therefore does not need an input.

```suggestion
```

Reply:
Just reverted this; thank you